pkg/controllers/disruption/controller.go

```diff
@@ -284,7 +284,7 @@ func (c *Controller) logAbnormalRuns(ctx context.Context) {
 func (c *Controller) logInvalidBudgets(ctx context.Context) {
 	nodePoolList := &v1beta1.NodePoolList{}
 	if err := c.kubeClient.List(ctx, nodePoolList); err != nil {
-		logging.FromContext(ctx).Errorf("listing nodepools, %s", err)
+		log.FromContext(ctx).Error(fmt.Errorf("listing nodepools, %w", err), "Disruption error")
```

Comment:
Can we decapitalize to match all the other log formats: 

```suggestion
		log.FromContext(ctx).Error(fmt.Errorf("listing nodepools, %w", err), "disruption error")
```

This is what it currently looks like. 

```
karpenter-6d49c99994-njcg4 controller {"level":"ERROR","time":"2024-05-20T17:04:56.019Z","logger":"controller.disruption","message":"Disruption error","commit":"40d7cf0-dirty","error":"detected disruption budget errors, invalid disruption budgets in nodepool default, invariant violated, invalid cron %!s(<nil>)"}
```